tests/cover/test_line.py

```diff
@@ -0,0 +1,9 @@
+from src.cover.cover import Line, Cover
+
+def test_line():
+    filepath = "/path/to/python/test.py"
+    line_no = 112
+    line_text = "def foo():\n"
+    line = Line(filepath, line_no, line_text)
+    assert str(line) ==  line_text[:-1]
+    assert int(line) == line_no
```

Comment:
```suggestion
    assert int(line) == line_no
    
```
Requires newline at the end.